README.md

```diff
@@ -63,17 +63,29 @@ You can collect them like this:
 Ferm::Rule <<| tag == 'allow_kafka_server2server' |>>
 ```
 
-You can also define rules in hiera:
+You can also define rules in Hiera. Make sure to use `alias` as interpolation function.
```

Comment:
Maybe add a statement that `alias()` is required because it returns the same datatype that the looked up key has. `hiera()` always returns a string.

Reply:
done